core/kernel/dt_driver_test.c

```diff
@@ -249,6 +258,43 @@ static TEE_Result dt_test_consumer_probe(const void *fdt, int node,
 		dt_test_state.probe_resets = SUCCESS;
 	}
 
+	if (IS_ENABLED(CFG_DRIVERS_GPIO)) {
+		struct gpio *gpio = NULL;
+
+		DT_TEST_MSG("Probe GPIO controllers");
+
+		res = gpio_dt_get_by_index(fdt, node, 0, "foo", &gpio);
```

Comment:
I think `"foo"` should be replaced with `"test"`.
Here and at line 281.

Please [build and run platform `qemu_virt`](https://optee.readthedocs.io/en/latest/building/devices/qemu.html#build-instructions) with this series applied. This GPIO drivers test should be excercised and return successfully: check the result of `xtest regression_1001`.